Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -3173,18 +3216,241 @@ def fetch_incidents():
             current_fetch_ofp_detection,
         ]
     )
-    return (
-        incidents
+    return (incidents
         + detections
         + idp_detections
         + iom_incidents
         + ioa_incidents
         + mobile_detections
         + on_demand_detections
-        + ofp_detections
-    )
+        + ofp_detections)
+
+def fetch_events():
```

Comment:
as part of the refactor we will break this very long method to smaller methods and unify the same logic from fetch_incidents to a single method to prevent code duplication